src/commonMain/kotlin/fr/acinq/lightning/wire/LightningMessages.kt

```diff
@@ -1840,3 +1834,13 @@ data class OnionMessage(
         }
     }
 }
+
+data class UnknownMessage(
+    override val type: Long
+
+) : LightningMessage {
+    override fun write(out: Output) {
+        TODO("Serialization of unknown messages is not implemented")
```

Comment:
We don't need that right?

Reply:
No I don't see a use-case where we would need to persist unknown LN messages.